modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-respect-rbac-using-gitops_{context}"]
+= Configuring respectRBAC using {gitops-title}
+
+The `respectRBAC` configuration feature in Argo CD controls how Argo CD watches resources on a cluster. By default, Argo CD attempts to watch all Kubernetes resources (CRDs) on a cluster at the cluster scope. You can restrict the ArgoCD controller from discovering or syncing specific resources using only controller RBAC, without manually configuring resource exclusions.
```

Comment:
`By default, Argo CD attempts (...). You can restrict the ArgoCD controller from discovering`

With this wording, it is not clear that the second sentence (Beginning with 'You can') is referencing the new feature. As worded, it could also be referencing the default from the previous sentence.

An alternative would be `By default, Argo CD attempts (...). With the respectRBAC feature, you can...`

Reply:
Fixed.